Makefile

```diff
@@ -18,7 +18,7 @@ HELM_IMAGE_REPO_SERVER ?= quay.io/kiali/kiali
 DORP ?= docker
 
 # When building the helm chart, this is the helm version to use
-HELM_VERSION ?= v3.5.2
+HELM_VERSION ?= v3.10.1
```

Comment:
is this bump of helm required for ARM support? If not, it would not be related to this issue and so should not be included.

Reply:
Yeah it is, as ARM support started from 3.6.0. and if to bump it, why not to bump to the latest.
Let me know what do you think